docs/website/docs/plus/core-concepts/project.md

```diff
@@ -1,163 +1,33 @@
-
-#### Profiles
-
-You can use the profiles section to define different environments (example: dev, staging, prod, tests). One package may have multiple profiles which can be specified using dlt+ CLI commands. The default profile name is `dev`. If you want to use dlt test helpers, you must define a `tests` profile as well.
-
-```yaml
-profiles:
-    dev: # Using "dev" profile will write to local filesystem
-        destinations:
-            delta_lake:
-                type: filesystem
-                bucket_url: ${tmp_dir}delta_lake
-    prod: # Using "prod" profle will write to an s3 bucktet
-        destinations:
-            delta_lake:
-                type: filesystem
-                bucket_url: s3://dlt-ci-test-bucket/dlt_example_package/
-
-```
-
-#### Misc settings
-
-It is also possible to add additional dlt settings that mirror the `config.toml` settings:
-
-```yaml
-runtime:
-  log_level: WARNING
-```
-
-### Config and secrets
-
-As shown above, it is possible to pass additional dlt settings and configurations in the manifest file itself. But existing dlt config providers are also supported as usual, like:
-
-1. environ provider
-2. `.dlt/config.toml` provider, including the global config
-3. `.dlt/<profile_name>.secrets.toml`, which is the secrets toml provider but scoped to a particular profile. A per-profile version (`dev.secrets.toml`) is sought instead of the `secrets.toml` file.
-
-### Python interface to use and share the data
-
-dlt packages provide a Python interface to access the data. This is done by exposing a standard Python API which provides connection to the production data through a profile called "access". The access to data can be limited by applying data and schema contracts on top of dlt datasets.
-
-### Configuration precedence
-Based on the information about precedence in the [configuration docs](../../general-usage/credentials/setup#available-config-providers), the yaml files provide the lowest precedence of all providers just above the default values for a config value. Settings in the yaml file will therefore be overridden by `toml` and `env` variables if present.
+├── .dlt/                 # your dlt settings including profile settings
+│   ├── config.toml
+│   ├── dev.secrets.toml
+│   └── secrets.toml
+├── _storage/             # local storage for your project, excluded from git
+├── sources/              # your sources, contains the code for the arrow source
+│   └── arrow.py
+├── .gitignore
+├── requirements.txt
+└── dlt.yml       # the main project manifest
+```
+
+dlt+ Project follows the standard Python package structure, allowing users to package their code and configuration as Python packages. These packages can be distributed via PyPI (private or open-source) or as a git repository. Any Python project manager can be used for packaging such as [uv](https://docs.astral.sh/uv/) or [poetry](https://python-poetry.org/).
+
+Read more about dlt+ Project in [Projects section](../features/projects.md)  
```

Comment:
I know I put it here, but feels weird. Here is another suggestion. What do you think?

```suggestion
Read more about dlt+ Project in the [project feature description](../features/projects.md)  
```